### PR TITLE
Remove duplication

### PR DESCRIPTION
**Pull request by mooncak**
Remove duplication in below files:

1. contributors/design-proposals/scheduling/taint-toleration-dedicated.md
2. contributors/design-proposals/storage/container-storage-interface.md
3. contributors/design-proposals/storage/volume-snapshotting.md
4. contributors/design-proposals/storage/volume-topology-scheduling.md

### CODE REVIEW

File: contributors/design-proposals/resource-management/admission_control_resource_quota.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ $ kubectl create -f test/fixtures/doc-yaml/admin/resourcequota/namespace.yaml
 namespace "quota-example" created
 $ kubectl create -f test/fixtures/doc-yaml/admin/resourcequota/quota.yaml --namespace=quota-example
 resourcequota "quota" created
-$ kubectl describe quota quota --namespace=quota-example
```
Comment (cblecker) [inline contributors/design-proposals/resource-management/admission_control_resource_quota.md]: This is intended. Please revert.

Comment (mooncak) [reply contributors/design-proposals/resource-management/admission_control_resource_quota.md]: reverted

**Comment by mooncak** [comment]
@cblecker  reverted admission_control_resource_quota.md

**Pull request merged**